docs/docs/getting-started/using-your-kernel.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 4
+title: Using your Kernel
+---
+
+Currently, kernels are only usable on Noteable if they are built by [noteable-io/kernels](https://github.com/noteable-io/kernels), or
+built using `Custom Project Kernels`. We don't currently support using custom images built by external systems.
```

Comment:
```suggestion
. We don't currently support using custom images built by external systems.
```

I don't think we need to mention that part unless we have docs to talk about that